src/javascript/app/pages/user/account/statement/statement.init.js

```diff
@@ -53,11 +53,13 @@ const StatementInit = (() => {
     };
 
     const getAccountStatistics = () => {
-        // only show Account Statistics to MLT/MX clients
-        if (!/^MLT|MX/.test(Client.get('loginid'))) return;
+        BinarySocket.wait('landing_company').then(() => {
+            // only show Account Statistics to MLT/MX clients
+            if (!/malta|iom/.test(Client.get('landing_company_shortcode'))) return;
```

Comment:
afaik we get this value from authorize, so no need to wait for landing company

Reply:
and for malta you need to test exactly malta otherwise it will also be true for maltainvest. something like `/^(malta|iom)$/` should do